src/metpy/io/__init__.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2015,2016,2018 MetPy Developers.
+# Copyright (c) 2015,2016,2018,2021 MetPy Developers.
```

Comment:
I'm not sure what GitHub's deal is here, but this diff actually looks fine on the command line.